packages/tables/src/Table/Concerns/CanScrollToTop.php

```diff
@@ -0,0 +1,23 @@
+<?php
+
+namespace Filament\Tables\Table\Concerns;
+
+use Closure;
+
+trait CanScrollToTop
+{
+    protected bool | Closure $shouldScrollToTopOnPageChange = false;
+
+    public function scrollToTop(bool | Closure $condition = true): static
```

Comment:
I think we should keep the same name for the method:

```suggestion
    public function scrollToTopOnPageChange(bool | Closure $condition = true): static
```